src/components/dashboard/sidebar/SubsidiesSummary.jsx

```diff
@@ -77,7 +75,47 @@ const SubsidiesSummary = ({
 
   return (
     <>
-      {hasAssignedSubsidyOrRequests && (
+      {hasAssignedOfferAndCodes && !hasAssignedSubsidyAndOffer && (
```

Comment:
@adamstankiewicz I'm wondering about the variable names in this file. A number of these variables have conditionals baked in to the name:
- `hasAssignedOfferAndCodes`
- `hasAssignedSubsidyAndOffer`
- `hasActiveLicenseOrLicenseRequest`
- etc...

It seems a little weird? Then when you come down to this part of the file to evaluate what should be displayed you end up reading this odd mix of variables with logical operators in their name mixed with the symbolic logical operators and my ADD is triggered.  

Perhaps even if we just calculated this boolean algebra all together in the file with better documentation it would be ok? Curious to hear folks thoughts.
Could this be worthy of a refactor?